docs/SetupGuide_DotnetOutOfProc.md

```diff
@@ -66,7 +66,7 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
 
     ```csharp
     [Function("GetEmployees")]
-    public static async IEnumerable<Employee> Run(
+    public static async Task<IEnumerable<Employee>> Run(
```

Comment:
I'd just remove the async. This doesn't need to be async the way it's written currently and we want these to be as simple as possible.